### PR TITLE
A few ideas for adding to the static landing page

### PR DESCRIPTION
**Pull request by wadefagen**
Largely kept the structure @nwalters512 created, re-worded a few things, added a few more sections, created a grid below the header container, and reduced the font in several places to get more "above the fold" on mobile.

Still need to add the poem back somewhere. :(

Phone Screen "Above the Fold":
image

Desktop Screen:
image

### CODE REVIEW

**Comment by nwalters512** [comment]
Gonna check this out! Is this designed to supersede #6?

File: package-lock.json [hunk 1]
```diff
@@ -9,115 +9,115 @@
       "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.0.0.tgz",
       "integrity": "sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==",
       "requires": {
-        "@babel/highlight": "^7.0.0"
+        "@babel/highlight": "7.0.0"
```
Comment (nwalters512) [inline package-lock.json:12]: Can you check which version of npm you're using? This PR shouldn't need to change the lockfile.

Comment (wadefagen) [reply package-lock.json:12]: probably an old one :P

**Comment by wadefagen** [comment]
Merging with #6 will add there

**Pull request closed without merging**